uwblueprint/child-witness-centre
# Set up MongoDB data seeding

## Notion Ticket Link

[Setup Database Seeding](https://www.notion.so/uwblueprintexecs/Setup-Database-Seeding-3c53b6092f7547eda00950b482d9fb2b)



## Implementation Description
* Created two TypeScript scripts to seed user and entity data into MongoDB database.



## Steps to Test
1. Run `npx ts-node backend/seeding/seedEntities.ts` or `npx ts-node backend/seeding/seedUsers.ts` to create 10 random entities or users.
2. Open a terminal and type `mongosh mongodb+srv://cwc:jEkj8yE3mOWeviIO@child-witness-centre.6c4i8ne.mongodb.net/` to connect to the database, then type `db.users.find().sort([['_id', -1]] )` to check the latest entries.



## What should reviewers focus on?
* 


## Screenshots (optional)


## Checklist

## Linting & Formatting
### Mac
```bash
docker exec -it CWC-backend /bin/bash -c "yarn fix"
docker exec -it CWC-frontend /bin/bash -c "yarn fix"
```

### Windows
```bash
docker exec -it CWC-backend bash -c "yarn fix"
docker exec -it CWC-frontend bash -c "yarn fix"
```
